cronus/deployment.yaml.in

```diff
@@ -0,0 +1,37 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: cronus-deployment
+  labels:
+    app: cronus
+    hail.is/sha: "@sha@"
+spec:
+  selector:
+    matchLabels:
+      app: cronus
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: cronus
+        hail.is/sha: "@sha@"
+    spec:
+      containers:
+      - name: cronus
+        image: @image@
```

Comment:
Why is `@sha@` in double quotes, but not `@image@`?

Reply:
YAML is another syntax for JavaScript.  If you write a string that looks like a number (e.g. 123) YAML will make it into a number.  If you actually want it to be a string, you need to wrap it in quotes.  image starts with gcr.io/... so it can never be a number (although quotes wouldn't hurt). The sha (with very small probability) can, so this is correct.  I'm not sure we do this everywhere, but we should.

Reply:
I actually got burned by the lack of quotes on "9978120". I have a forthcoming PR to double-quotify all SHA replacements.

Reply:
And here's the PR: https://github.com/hail-is/hail/pull/4580